dftimewolf/cli/dftimewolf_recipes.py

```diff
@@ -292,6 +292,20 @@ def CleanUpPreflights(self) -> None:
     """Calls the preflight's CleanUp functions."""
     self._state.CleanUpPreflights()
 
+  def PrintStats(self) -> None:
+    """Prints collected stats if existing."""
+    statentries = self._state.GetStats()
```

Comment:
Nit: `state_entries` instead? Because I first read this as `state_ntries`

Reply:
Great suggestion, done.